CMakeLists.txt

```diff
@@ -790,6 +791,14 @@ IF (USE_VTUNE_API)
   LINK_LIBRARIES("${VTUNE_ITTNOTIFY_LIBRARY}")
 ENDIF()
 
+SET(USE_NVTX_API 0 CACHE BOOL "Enable/disable NVTX regions in CUDA code.")
+IF(USE_NVTX_API)
+  IF(HAVE_CUDA)
+    LINK_DIRECTORIES("${CUDA_TOOLKIT_ROOT_DIR}/lib64")
```

Comment:
I still have the issue that this link directory is not correct for all platforms.  I will fix.